source/ceylon/time/Date.ceylon

```diff
@@ -32,6 +32,15 @@ shared interface Date
 
     "Returns the [[DateRange]] between this and given [[Date]]."
     shared formal DateRange rangeTo( Date other );
+    
+    "Two `Date`s are considered equals if they have the same 
```

Comment:
Here I tried to explain _how_ two dates are considered the same because I think this was the reason to explicit enforce equals in the interface, but I can change to your comment if you want.. let me know